raiden/messages.py

```diff
@@ -597,39 +604,20 @@ def from_event(cls, event):
 
 
 @dataclass(repr=False, eq=False)
-class Withdraw(SignedRetrieableMessage):
-    """ Confirms withdraw to partner with a signature """
+class WithdrawRequest(WithdrawBase):
+    """ Requests a signed on-chain withdraw confirmation from partner. """
 
-    cmdid: ClassVar[int] = messages.WITHDRAW
+    pass
 
-    chain_id: ChainID
-    token_network_address: TokenNetworkAddress
-    channel_identifier: ChannelID
-    participant: Address
-    total_withdraw: WithdrawAmount
-    nonce: Nonce
 
-    def pack(self, packed):
-        packed.chain_id = self.chain_id
-        packed.token_network_address = self.token_network_address
-        packed.channel_identifier = self.channel_identifier
-        packed.total_withdraw = self.total_withdraw
-        packed.participant = self.participant
-        packed.message_type = MessageTypeId.WITHDRAW
-        packed.signature = self.signature
+@dataclass(repr=False, eq=False)
+class Withdraw(WithdrawBase):
```

Comment:
could you please rename this to `WithdrawConfirmation`? To match the state machine code.

Reply:
Done